src/emqtt_bench.erl

```diff
@@ -345,7 +359,7 @@ start(PubSub, Opts) ->
                    ConnRate ->
                        1000 * NoWorkers div ConnRate
                end,
-
+    PayloadHdrs = string:tokens(proplists:get_value(payload_hdrs, Opts, []), ","),
```

Comment:
Nit: maybe we could parse the arguments to avoid crashing later (i.e.: check is it's one of the allowed options)?